src/main/java/core/basesyntax/model/Purchase.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.model;
+
+public final class Purchase extends Operation {
```

Comment:
I suggest making Operation enum, operation objects always should be the same and in limited amount. And create FruitTransaction class to keep operation, amount and fruit name